InfoMovies/InfoMovies/Views/ItemDetailsPage.xaml

```diff
@@ -0,0 +1,127 @@
+                                                                        LineBreakMode="NoWrap" VerticalTextAlignment="Center" FontSize="Micro"/>
+                                                </StackLayout>
+
+                                            </StackLayout>
+                                        </DataTemplate>
+                                    </Views:HorizontalListView.ItemTemplate>
+                                </Views:HorizontalListView>
+                            </ScrollView>
+
+                        </StackLayout>
+                    </Grid>
+
+                    <ScrollView Orientation="Horizontal" BindingContext="{Binding Credits}" Style="{StaticResource scrollViewStyle}">
+                        <Views:HorizontalListView Style="{StaticResource gridStyle}" BackgroundColor="DarkGray" ItemsSource="{Binding CastCollection}">
+                            <Views:HorizontalListView.ItemTemplate>
+                                <DataTemplate>
+                                    <StackLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"
+                                                 Padding="10" Orientation="Vertical">
+                                        <Grid Style="{StaticResource gridStyle}" AutomationId="MovieItem">
+                                            <Grid.RowDefinitions>
+                                                <RowDefinition Height="150"/>
+                                            </Grid.RowDefinitions>
+                                            <Grid.ColumnDefinitions>
+                                                <ColumnDefinition Width="*"/>
+                                            </Grid.ColumnDefinitions>
+                                            <BoxView Color="Black" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"/>
+                                            <ImageLoading:CachedImage HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"
+                                                                Aspect="AspectFit" Source="{Binding Path=ProfilePath}"/>
+                                        </Grid>
+                                        
+                                        <StackLayout Orientation="Horizontal" Padding="0" Style="{StaticResource stackLayoutStyle}">
+                                            <Label Style="{StaticResource labelStyle}" Text="{Extensions:Translate Text=ItemDetailsPage_Name}"
+                                                   LineBreakMode="NoWrap" TextColor="Black" VerticalTextAlignment="Center"
+                                                   HorizontalOptions="Start" FontSize="Micro"/>
+                                            <Label Style="{StaticResource labelStyle}" Text="{Binding Name}" LineBreakMode="NoWrap"
+                                                   TextColor="Black" VerticalTextAlignment="Center" HorizontalOptions="End" FontSize="Micro"/>
```

Comment:
How could you improve the use of Styles, for example lableStyle?

Reply:
creating inherited styles according to the repetition of controls with similar appearance
I have created several labels repeating some properties. :(